js/models/conversations.js

```diff
@@ -1191,6 +1191,18 @@
       };
     },
 
+    getOpenGroup() {
```

Comment:
```suggestion
    toOpenGroup() {
```

Reply:
Hmm i'm 50/50 on this name change, I'll wait for others to comment on it. @msgmaxim @vincentbavitz

Reply:
I'm 60/40 in favour of `toOpenGroup` :)

Reply:
Changed :)